pkg/router/router.go

```diff
@@ -46,7 +46,7 @@ func registerWeb(resource endpoints.Resource, mux *http.ServeMux) {
 			w.Header().Set("Cache-Control", "public, max-age=31536000, immutable")
 		}
 
-		w.Header().Set("Content-Security-Policy", "default-src 'none'; img-src 'self'; script-src 'self'; style-src 'self'; connect-src 'self' wss: ws:; font-src 'self' https://1.www.s81c.com;")
+		w.Header().Set("Content-Security-Policy", "default-src 'none'; img-src 'self'; script-src 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; connect-src 'self' wss: ws:; font-src 'self' https://1.www.s81c.com;")
```

Comment:
Do we really need `script-src 'unsafe-eval'` for prod?
It was only used in the webpack dev config due to the hot reloading mechanism but I wouldn't have expected CodeMirror to be using eval itself

Reply:
yeah, tested, they not needed